sdk-python/tests/test_workflow.py

```diff
@@ -1160,6 +1166,7 @@ def my_entrypoint(thread: WorkflowThread) -> None:
         self.assertEqual(
             wf.compile(),
             PutWfSpecRequest(
+                allowed_updates="ALL",
```

Comment:
As per the comment above, if you don't set this field, it defaults to `ALL`, so there's not really any need to set it.